compiler/dyno/lib/uast/chpl-syntax-printer.cpp

```diff
@@ -0,0 +1,1065 @@
+/*
+ * Copyright 2021-2022 Hewlett Packard Enterprise Development LP
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+#include "chpl/uast/chpl-syntax-printer.h"
+
+
+
+namespace chpl {
```

Comment:
Technically the only thing that needs to be in the `chpl` namespace here is the `printAst` function. I think we should make this namespace anonymous (with all the `ChplSyntaxVisitor` stuff in it` then have a small `namespace chpl { ...` block at the bottom. This is similar to what is done for `compiler/passes/convert-uast.cpp` if you need a concrete example. See `convertToplevelModule`.